x/oracle/client/cli/txApproveOracleRegistration.go

```diff
@@ -0,0 +1,43 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdApproveOracleRegistration() *cobra.Command {
```

Comment:
I think this CLI is not essential, but I'm not against to have it, as long as the panacea-oracle provides a CLI for generating a signature which is needed for this CLI.

Reply:
I see. There is no way to create a signature outside of oracle, and there is no reason to use this CLI. So, I remove this CLI for now.